tools/sql_extraction/src/main/kotlin/App.kt

```diff
@@ -0,0 +1,9 @@
+package com.google.cloud.sqlextraction
```

Comment:
Have you checked with Zihao to see if this the namespace to use?

Reply:
Ah, I missed sqlecosystem. I can fix that.